### ISSUE TITLE
False positives for attribute selectors in indentation

### ISSUE THREAD

**Issue by HauptmannEck** [stylelint/stylelint#3156]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug in indetation linting for selector list with bracket selectors. The demo page does not show this as an warning.

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```css
.a > div,
.b > div[type=text] {
    color: blue;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "indentation": 4
  }
}
```

> Which version of stylelint are you using?

`8.4.0`
Works in version:
`7.13.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint "src/**/*.less" `

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
src/static/test.less
 2:1  ✖  Expected indentation of 4 spaces   indentation
```

**Comment by evilebottnawi** [comment]
@HauptmannEck Thanks for issue, seems problem in less parser, not in stylelint.

**Comment by jeddy3** [comment]
Closing as dupe of https://github.com/stylelint/stylelint/issues/2791

@HauptmannEck Please consider contributing a fix to the LESS parser via https://github.com/stylelint/stylelint/issues/2791 as I don't believe many of the core team use Less (or preprocessors) so its support is reliant on outside contributors.
